crates/nu-command/src/progress_bar.rs

```diff
@@ -0,0 +1,71 @@
+use indicatif::{ProgressBar, ProgressState, ProgressStyle};
+use std::fmt;
+
+// This module includes the progress bar used to show the progress when using the command `save`
+// Eventually it would be nice to find a better plece for it.
+
+pub struct NuProgressBar {
+    pub pb: ProgressBar,
+    bytes_downloaded: u64,

```

Comment:
```suggestion
    bytes_processed: u64,
```

`bytes_downloaded` feels like the wrong name because the bytes aren't always downloaded (maybe they're loaded from elsewhere on disk)